### DISCUSSION TITLE
Installation step by step , Nginx, Redis, PostgreSQL

### DISCUSSION THREAD

**Discussion by weasam** [mediacms-io/mediacms#715] [General]
MediaCMS is a wonderful project. but for some user they may have a machine has already installed Nginx, Redis, PostgreSQL, that need to install MediaCMS individually instead by use the script. I check the install script is the commands for install dependencies. 

has anyone installed success step by step, I don't want to buy a new VPS or create a new virtual machine.

the docker installation need to be improve, Docker version of mediacms required PostgreSQL Redis. but my server already installed these which the ports already been using, any way to configure to different port for PostgreSQL and Redis?

**Comment by mgogoulos** [comment]
Hi, it's easy to get a new environment these days (on any cloud service) and try it out, or even provision a new VM on your local infrastructure, and this is what is recommended rather than installing on a server that currently is running any of these services (redis/postgresql/nginx...)

**Reply by weasam** [reply]
Thanks for your reply
I had got peertube installed on my raspberry PI. all install separately instead of the script installation.  So at this moment is better to have a clean environment to install mediaDC

**Comment by gevorg-vardanyan-im** [comment]
Hi all, I have a similar question.
I need to start development server locally and do some changes of the code.
It is possible to install server via 'install.sh' script or via docker, it worked on my machine.
But this is not appropriate option to continue local development.
I have tried to run server with these commands:
1. python manage.py makemigrations
2. python manage.py migrate
3. python manage.py collectstatic
4. python manage.py runserver
Server started successfully, but static files are not loaded, and I can't access to main page.
Admin page is accessible, but without styles.

I have tried to run frontend with this command:
1. npm install
But this command stucks with this message: 
WARN deprecated core-js@2.6.12: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues.

Could you please help me to run server locally and start development ?

Thank you in advance.
